domain/schema/controller/sql/0022-model-authorized-keys.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE model_authorized_keys (
+    model_uuid TEXT NOT NULL,
+    user_public_ssh_key_id INTEGER NOT NULL,
+    PRIMARY KEY (model_uuid, user_public_ssh_key_id),
+    FOREIGN KEY (user_public_ssh_key_id)
+    REFERENCES user_public_ssh_key (id),
+    FOREIGN KEY (model_uuid)
+    REFERENCES model (uuid)
+);
+
+--CREATE UNIQUE INDEX idx_model_authorized_keys_composite
+--ON model_authorized_keys (model_uuid, user_public_ssh_key_id);
+
+CREATE INDEX idx_model_authorized_keys_model_uuid
+ON model_authorized_keys (model_uuid);
+
+CREATE INDEX idx_model_authorized_keys_model_uuid_user_public_ssh_key_id
+ON model_authorized_keys (user_public_ssh_key_id);
+
+-- v_model_authorized_keys provides a nice view of what public ssh keys are
+-- currently allowed on a model making sure that users that are removed and or
+-- disabled have thier authorized keys removed from the model.
+CREATE VIEW v_model_authorized_keys AS
+SELECT
+    mak.model_uuid,
+    upsk.public_key
+FROM model_authorized_keys AS mak
+INNER JOIN user_public_ssh_key AS upsk ON mak.user_public_ssh_key_id = upsk.id
+INNER JOIN user AS u ON upsk.user_uuid = u.uuid
```

Comment:
We're not quite consistent across the code base - I thought we were preferring just "JOIN". Something to follow up on and fix later if needed.

Reply:
I haven't seen this. Can you point me to where we are doing this and i'll move over.